From 0edf3d2eff88d7594ed7dc070f889b1bcacb677f Mon Sep 17 00:00:00 2001
From: "Tobias C. Berner" <tcberner@FreeBSD.org>
Date: Sun, 24 Jun 2018 08:25:53 +0200
Subject: [PATCH] CPP-616 - Use uv/version.h as version file for libuv v1.21.0+

libuv v1.21.0+ changed where the version file is located.

Patch-Source: https://github.com/datastax/cpp-driver/commit/0edf3d2eff88d7594ed7dc070f889b1bcacb677f
---
 cmake/modules/CppDriver.cmake | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/cmake/modules/CppDriver.cmake b/cmake/modules/CppDriver.cmake
index 92c243d5..098374f9 100644
--- a/cmake/modules/CppDriver.cmake
+++ b/cmake/modules/CppDriver.cmake
@@ -372,6 +372,8 @@ macro(CassUseLibuv)
 
   if (EXISTS "${LIBUV_INCLUDE_DIR}/uv-version.h")
     set(LIBUV_VERSION_HEADER_FILE "${LIBUV_INCLUDE_DIR}/uv-version.h")
+  elseif (EXISTS "${LIBUV_INCLUDE_DIR}/uv/version.h")
+    set(LIBUV_VERSION_HEADER_FILE "${LIBUV_INCLUDE_DIR}/uv/version.h")
   else()
     set(LIBUV_VERSION_HEADER_FILE "${LIBUV_INCLUDE_DIR}/uv.h")
   endif()
